server/routes/user.js

```diff
@@ -31,6 +31,10 @@ router.route('/:userId/device/:deviceId')
 
   .delete(validate(paramValidation.deviceById), userCtrl.removeDevice);
 
+router.route('/:userId/group')
```

Comment:
Should this be `/groups`? because it's multiple in this case. i'm not sold on either way yet